test/integration/auth/node_test.go

```diff
@@ -275,134 +274,156 @@ func TestNodeAuthorizer(t *testing.T) {
-	expectAllowed(t, deleteNode2NormalPod(node2Client))
-	expectAllowed(t, createNode2MirrorPod(node2Client))
-	expectAllowed(t, deleteNode2MirrorPod(node2Client))
+	expectAllowed(t, getSecret, node2Client)
+	expectAllowed(t, getPVSecret, node2Client)
+	expectAllowed(t, getConfigMap, node2Client)
+	expectAllowed(t, getPVC, node2Client)
+	expectAllowed(t, getPV, node2Client)
+
+	expectForbidden(t, createNode2NormalPod, node2Client)
+	expectAllowed(t, updateNode2NormalPodStatus, node2Client)
+	expectAllowed(t, deleteNode2NormalPod, node2Client)
+	expectAllowed(t, createNode2MirrorPod, node2Client)
+	expectAllowed(t, deleteNode2MirrorPod, node2Client)
 	// recreate as an admin to test eviction
-	expectAllowed(t, createNode2NormalPod(superuserClient))
-	expectAllowed(t, createNode2MirrorPod(superuserClient))
-	expectAllowed(t, createNode2NormalPodEviction(node2Client))
-	expectAllowed(t, createNode2MirrorPodEviction(node2Client))
+	expectAllowed(t, createNode2NormalPod, superuserClient)
+	expectAllowed(t, createNode2MirrorPod, superuserClient)
+	expectAllowed(t, createNode2NormalPodEviction, node2Client)
+	expectAllowed(t, createNode2MirrorPodEviction, node2Client)
 
 	// re-create a pod as an admin to add object references
-	expectAllowed(t, createNode2NormalPod(superuserClient))
+	expectAllowed(t, createNode2NormalPod, superuserClient)
 	// With ExpandPersistentVolumes feature disabled
 	defer utilfeaturetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.ExpandPersistentVolumes, false)()
 	// node->pvc relationship not established
-	expectForbidden(t, updatePVCCapacity(node1Client))
+	expectForbidden(t, updatePVCCapacity, node1Client)
 	// node->pvc relationship established but feature is disabled
-	expectForbidden(t, updatePVCCapacity(node2Client))
+	expectForbidden(t, updatePVCCapacity, node2Client)
 
 	//Enabled ExpandPersistentVolumes feature
 	defer utilfeaturetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.ExpandPersistentVolumes, true)()
 	// Node->pvc relationship not established
-	expectForbidden(t, updatePVCCapacity(node1Client))
+	expectForbidden(t, updatePVCCapacity, node1Client)
 	// node->pvc relationship established and feature is enabled
-	expectAllowed(t, updatePVCCapacity(node2Client))
+	expectAllowed(t, updatePVCCapacity, node2Client)
 	// node->pvc relationship established but updating phase
-	expectForbidden(t, updatePVCPhase(node2Client))
+	expectForbidden(t, updatePVCPhase, node2Client)
 }
 
-func expectForbidden(t *testing.T, err error) {
-	if !errors.IsForbidden(err) {
-		_, file, line, _ := runtime.Caller(1)
-		t.Errorf("%s:%d: Expected forbidden error, got %v", filepath.Base(file), line, err)
+// Node authorizer is watch-cache based, not instantaneous. Retry unexpected cases
+// a few times to ensure the authorizer hasn't just missed our event.
+var backoff = wait.Backoff{
+	Duration: time.Millisecond * 10,
+	Factor:   5,
+	Steps:    3,
```

Comment:
Consider steps: 4? That's a max wait of ~1.5s, vs ~300ms-- might be less flaky.

Reply:
But this is no retry => some retry, so you can bump up steps later if necessary.